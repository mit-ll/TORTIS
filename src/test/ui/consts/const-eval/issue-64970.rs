// run-pass

fn main() {
    foo(10);
}

fn foo(mut n: i32) {
    if false {
        n = 0i32;
    }

    if n > 0i32 {
        1i32 / n;
    }
}
