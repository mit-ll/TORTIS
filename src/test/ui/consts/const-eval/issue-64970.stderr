warning: unused arithmetic operation that must be used
  --> $DIR/issue-64970.rs:13:9
   |
LL |         1i32 / n;
   |         ^^^^^^^^
   |
   = note: `#[warn(unused_must_use)]` on by default

